client/app/cdn/dedicated/domain/add/cdn-dedicated-domain-add.html

```diff
@@ -1,91 +1,87 @@
-<div data-ng-controller="CdnAddDomainsCtrl">
+<div data-wizard
```

Comment:
Only previous inline `data-ng-controller` has been changed in this file (+ adding $uibModalInstance.dismiss)